clang/include/clang/Basic/SimpleTypoCorrection.h

```diff
@@ -0,0 +1,35 @@
+#ifndef LLVM_CLANG_BASIC_SIMPLETYPOCORRECTION_H
```

Comment:
Pretty sure this still needs the copyright header, even though it can be a simplified one these days.